src/containers/settings-bar.js

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { setSearchMethod, setCategorySearch, setCategoryValue } from '../actions/index';
+import { autobind } from 'core-decorators';
+
+class SettingsBar extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      long: {
+        isChecked: true
+      },
+
+      short: {
+        isChecked: false
+      },
+
+      categorySearch: {
+        isChecked: false,
+        selectedCategory: 15
+      }
+    }
+
+    this.setFilter = this.setFilter.bind(this);
+  }
+
+  setFilter(event) {
+    const { target: { id } } = event;
+    if (event.target.id === "long") {
```

Comment:
rewrite this as the following:

if (id) {
  this.setState({ short: { isChecked: !this.state.short.isChecked } });
  this.setState({ long: { isChecked: !this.state.long.isChecked } });
  this.props.setSearchMethod(event.target.id);
}